src/scripts/twitch.ts

```diff
@@ -39,6 +39,7 @@ class Emotes {
     private static BTTV_ENDPOINT = "https://api.betterttv.net/2/emotes";
     private static BTTV_CHANNEL_ENDPOINT = "https://api.betterttv.net/2/channels/";
     private static BTTV_CHANNELS = ["forsenlol"];
+    private static CUSTOM_EMOTES = [{"KUPPA": "https://i.imgur.com/xqTpGLI.png",}];
```

Comment:
This doesn't need to be an array. In fact, I have a feeling that your code below won't work because its an array